lib/Zonemaster/Backend/Validator.pm

```diff
@@ -211,6 +210,102 @@ sub jsonrpc_method {
     return joi->string->regex( $JSONRPC_METHOD_RE );
 }
 
+sub formats {
+    my ( $rpcapi ) = @_;
+    return {
+        domain => sub { check_domain( $rpcapi, @_ ) },
+        language_tag => sub { check_language_tag( $rpcapi, @_ ) },
+        ip => sub { check_ip( $rpcapi, @_ ) },
+        profile => sub { check_profile( $rpcapi, @_ ) },
+    };
+}
+
+sub check_domain {
```

Comment:
Is the set of `check_*()` procedures intended to be public? In that case maybe add a POD section describing them. E.g. something like the section describing the untaint interface.

If it's intended to be private, maybe communiacte this intention by renaming them to `_check*()`?

Reply:
I have added documentation for these.